scripts/mc_log_autocorrelations.py

```diff
@@ -14,13 +14,17 @@
 import redis
 import sys
 
+import six.moves as sm
+
 from hera_mc import autocorrelations, mc
 
 
 # Yay hardcoded config settings:
 
 redis_host = 'redishost'
-antnums = xrange(128)
+
+sm.range
+antnums = sm.range(128)
```

Comment:
do we want this rogue `sm.range` call here?
also this will make antnums an iterator won't it? do we want `list(sm.range(128))`

Reply:
yes on both counts. Thanks!